nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -90,10 +90,12 @@ is" without express or implied warranty.
 #define RRXINERAMA_DISABLED_COND TRUE
 
 #ifdef PANORAMIX
+  #undef PANORAMIX_DISABLED_COND
   #define PANORAMIX_DISABLED_COND (noPanoramiXExtension || PanoramiXExtensionDisabledHack)
 #endif
 
 #ifdef RANDR
+  #undef RRXINERAMA_DISABLED_COND
```

Comment:
Oh, if we need the #undef we could also change that to 
```
#ifdef ..
   #define (...)
#else
  # define .. TRUE
#endif
```
and remove the upper define